Gruntfile.coffee

```diff
@@ -41,6 +48,12 @@ module.exports = (grunt) ->
     shell:
       webdriverManagerUpdate:
         command: "node_modules/grunt-protractor-runner/node_modules/.bin/webdriver-manager update"
+      generateApiDocs:
+        command: [
+          "node_modules/docha/bin/docha -p '.tmp/elementHelper.spec.js' -o 'docs/elementHelper.md' -e _"
```

Comment:
I mentioned them in the [description](https://github.com/ReadyTalk/cukefarm/blob/api_docs/README.md#cukefarm) in the README and provided a link there to just to the docs directory. Do you think I should elaborate on that?